include/bluetooth/services/ots.h

```diff
@@ -486,6 +471,27 @@ struct bt_ots_feat {
 /** @brief Opaque OTS instance. */
 struct bt_ots;
 
+/** @brief Descriptor for OTS object addition */
+struct bt_ots_obj_add_param {
+	/* Object size */
+	uint32_t size;
```

Comment:
I guess this was actually also missing from the original implementation, but if we do `/**` instead of `/*` then it is captured by doxygen as `/**` is shorthand for `/* @brief` for a single line, so this should be
```suggestion
	/** Object size */
	uint32_t size;
```
or 
```suggestion
	/* @brief Object size */
	uint32_t size;
```

Reply:
This comment also applies to the other fields below.